x-pack/docs/en/security/securing-communications/index.asciidoc

```diff
@@ -8,6 +8,8 @@ manipulation of the data, and attempts to gain access to the server and thus the
 files storing the data. Securing your nodes helps reduce the risk from
 network-based attacks.
 
+WARNING: A full description of TLS/SSL, PKI (Public Key Infrastructure) certificates, and Certificate Authority is beyond the scope of this document. This page assumes prior knowledge of TLS/SSL as well as access to valid certificates.
```

Comment:
I have found this kind of statements not helpful. I feel they are slightly intimidating and I'm not sure we should put them in the docs. I've even seen this type of statement ( we have this in other places, i.e. cloud saml/oidc setup ) being used in presentations as a testament to the difficulty and complexity of our docs and configuration. 

We aim to make the process as easy as possible so that even folks with minimum experience/knowledge can take a swing at configuring TLS (because they have no choice, TLS in transport is mandatory) and although the gist of the statement is true, stating it out loud doesn't buy us or the users much. Could we rephrase this to a note or a suggestion with a slightly more positive wording?

Reply:
One option is for us to actually write that _Introduction to TLS_ documentation and link to it here.
I don't expecially want to explain TLS to users, but I'd rather do that then say something to the effect of _If you don't already know this information then Elasticsearch isn't for you_.

Note, this isn't just true for TLS. We also have that issue with things like YAML (for config files) so it might be worth a separate conversation around how do we help new users learn the prereq knowledge to operate ES well.

Reply:
If we're not providing enough information for people to successfully secure the stack, that's a bug IMO. I think providing an overview of TLS in the context of the Elastic Stack would be worthwhile. We don't have to go into the details of *how* public key encryption works, but we could definitely do a better job of introducing what's required and how it all fits together. 

There have been a lot of improvements around security to make things easier, @lcawl added some tutorials to make it clearer what you need to do, and there's recently been a lot of reshuffling of this content. But some of the incremental changes that were made along the way made sense at the time, but haven't resulted in a very coherent story. I think it's due for more than minor cleanup.

Reply:
I will agree that this note doesn't solve the bigger problem, so I'm removing it from this PR.